docs/get-started/nextjs.md

```diff
@@ -0,0 +1,921 @@
+{
+  "presets": [
+    [
+      "next/babel",
+      {
+        "styled-jsx": {
+          "plugins": ["@styled-jsx/plugin-sass"]
+        }
+      }
+    ]
+  ]
+}
+```
+
+## Postcss
+
+Next.js lets you [customize postcss config](https://nextjs.org/docs/pages/building-your-application/configuring/post-css). Thus this framework will automatically handle your postcss config for you.
+
+This allows for cool things like zero config tailwindcss! (See [Next.js' example](https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss))
+
+## Absolute Imports
+
+[Absolute imports](https://nextjs.org/docs/pages/building-your-application/configuring/absolute-imports-and-module-aliases#absolute-imports) from the root directory are supported.
+
+```jsx
+// All good!
+import Button from 'components/button';
+// Also good!
+import styles from 'styles/HomePage.module.css';
+
+export default function HomePage() {
+  return (
+    <>
+      <h1 className={styles.title}>Hello World</h1>
+      <Button />
+    </>
+  );
+}
+```
+
+Also OK for global styles in `.storybook/preview.js|ts`!
+
+```js
+// .storybook/preview.js|ts
+
+import 'styles/globals.scss';
+
+// ...
+```
+
+## Runtime Config
+
+Next.js allows for [Runtime Configuration](https://nextjs.org/docs/pages/api-reference/next-config-js/runtime-configuration) which lets you import a handy `getConfig` function to get certain configuration defined in your `next.config.js` file at runtime.
+
+In the context of Storybook with this framework, you can expect Next.js's [Runtime Configuration](https://nextjs.org/docs/pages/api-reference/next-config-js/runtime-configuration) feature to work just fine.
+
+Note, because Storybook doesn't server render your components, your components will only see what they normally see on the client side (i.e. they won't see `serverRuntimeConfig` but will see `publicRuntimeConfig`).
+
+For example, consider the following Next.js config:
+
+```js
+// next.config.js
+module.exports = {
+  serverRuntimeConfig: {
+    mySecret: 'secret',
+    secondSecret: process.env.SECOND_SECRET, // Pass through env variables
+  },
+  publicRuntimeConfig: {
+    staticFolder: '/static',
+  },
+};
+```
+
+Calls to `getConfig` would return the following object when called within Storybook:
+
+```json
+{
+  "serverRuntimeConfig": {},
+  "publicRuntimeConfig": {
+    "staticFolder": "/static"
+  }
+}
+```
+
+## Custom Webpack Config
+
+Next.js comes with a lot of things for free out of the box like sass support, but sometimes you add [custom webpack config modifications to Next.js](https://nextjs.org/docs/pages/api-reference/next-config-js/webpack). This framework takes care of most of the webpack modifications you would want to add. If Next.js supports a feature out of the box, then that feature will work out of the box in Storybook. If Next.js doesn't support something out of the box, but makes it easy to configure, then this framework will do the same for that thing for Storybook.
```

Comment:
Can you check the capitalization of Sas, Webpack, and SVGR so that we can ensure it's accurate?